src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -1761,6 +1757,23 @@ public void testGetOrdersNoParameters() {
     assertEquals(3, resp.getBody().as(PurchaseOrders.class).getTotalRecords().intValue());
   }
 
+  @Test
+  public void testGetOrderPOLinesByPoId() {
+    logger.info("=== Test Get Orders lines - With empty query ===");
+
+    final Response resp = RestAssured
+      .with()
+        .header(X_OKAPI_URL)
+        .header(EXIST_CONFIG_X_OKAPI_TENANT_LIMIT_10)
+      .get(LINES_PATH + "?query=purchase_order_id==" + ORDER_ID_WITH_PO_LINES)
+        .then()
+         // .statusCode(200)
```

Comment:
why is this commented out?

Reply:
Just forgot to uncomment this after debugging. Fixed.